src/components/Routes/Routes.tsx

```diff
@@ -84,14 +85,18 @@ const Routes = () => {
         </Route>
       )}
       {authEnabled && <Route path="/login" element={<Login />} />}
-      <Route
-        path="*"
-        element={
-          <RequireAuth>
-            <Navigate replace to="/systems" />
-          </RequireAuth>
-        }
-      />
+      {isPermissionsSet === authEnabled || isPermissionsSet ? (
```

Comment:
The problem I have with using `isAuthenticated()` is the token gets set before the permissions do so the router tries to use the permissions before they are set. I made some changes to the permissions container that make it so `isPermissionsSet` will work as a function instead of a value, so it will not need to use the cookie.